tests/contracts/register/src/lib.rs

```diff
@@ -90,3 +92,21 @@ impl Apply<Gossip> for Register {
         }
     }
 }
+#[transaction]
+pub struct Unarchive;
+impl Transaction for Unarchive {
+    const NAME: &'static str = "unarchive";
+    type Return = ();
+}
+#[apply(name = "unarchive")]
+impl Apply<Unarchive> for Register {
+    fn apply(&mut self, _: Unarchive, _: StoreContext) {
+        /*
+        Unarchive all data in the state
+         */
+        let branch_mut = self.open_secrets.walk_mut(All).expect("Some(Branch)");
+        for leaf in branch_mut {
+            *leaf.value_mut() += 0;
```

Comment:
even though this is a no-op, and the compiler most likely does not optimize it away, using a + 1 would be more clear.

Reply:
the +1 would also unarchive, and the changes would not be "saved" but i guess it does not matter much, just for clarity